src/os/bluestore/NVMEDevice.cc

```diff
@@ -365,7 +365,12 @@ void SharedDriverQueueData::_aio_handle(Task *t, IOContext *ioc)
  again:
     dout(40) << __func__ << " polling" << dendl;
     if (current_queue_depth) {
-      spdk_nvme_qpair_process_completions(qpair, g_conf->bluestore_spdk_max_io_completion);
+      r = spdk_nvme_qpair_process_completions(qpair, g_conf->get_val<uint64_t>("bluestore_spdk_max_io_completion"));
```

Comment:
get_val will be a bit slow than before.  so the better idea is assign to a variable firstly, then uses the variable